README.md

```diff
@@ -1,2 +1,8 @@
 # passkeys-index
-A collection of sites and services that support passkeys
+Welcome to the passkey index, a community initiative to surface new websites and applications that support passkeys. This index is backed by security enthusiasts who want to share the newest and latest around passkeys.
+
```

Comment:
View the list here[https://github.com/bitwarden/passkeys-index/blob/main/passkey-index.md](url)